app/javascript/blacklight/modal.js

```diff
@@ -183,13 +177,6 @@ const Modal = (() => {
 
     // Catch our own custom loaded event to implement data-blacklight-modal=closed
     $('body').on('loaded.blacklight.blacklight-modal', Blacklight.modal.checkCloseModal);
-
-    // we support doing data-dismiss=modal on a <a> with a href for non-ajax
-    // use, we need to suppress following the a's href that's there for
-    // non-JS contexts.
-    $('body').on('click', Blacklight.modal.modalSelector + ' a[data-dismiss~=modal]', function (e) {
-      e.preventDefault();
-    });
```

Comment:
Does this not get use by this flash component? https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/system/flash_message_component.html.erb#L3

Reply:
nevermind